src/emailLogin/runMigration.js

```diff
@@ -0,0 +1,85 @@
+const axios = require('axios');
+const fs = require('fs');
+
+const { getDb } = require('../../db/index');
+
+const { PERSONAL_ACCESS_TOKEN, GITHUB_ORG_NAME: ISOMER_GITHUB_ORG_NAME } = process.env;
+
+const ISOMER_USERS = ['isomeradmin', 'rc-davis', 'lamkeewei', 'pallani', 'LoneRifle', 'prestonlimlianjie', 'alexanderleegs', 'lisatjide', 'kwajiehao', 'gweiying', 'seaerchin', 'isomer-demo', 'NatMaeTan', ' jacksonOGP', 'chienlinggg', 'kathleenkhy', 'joshuajunmingt', 'audreytcy', 'yanjunquek', 'chloe-opengovsg', 'shazlithebestie', 'lennardl', 'oliverli', 'taufiq'];
+
+// name of repo
+const REPO = process.argv[2];
+
+const getSiteAndContributors = async (site, dbClient) => {
+  try {
+    const { data: respData } = await axios.get(
+      `https://api.github.com/orgs/${ISOMER_GITHUB_ORG_NAME}/teams/${site}/members`,
+      {
+        headers: {
+          Authorization: `token ${PERSONAL_ACCESS_TOKEN}`,
+        },
+      },
+    );
+    if (!respData) {
+      return;
```

Comment:
should we log/throw some error here? This would be unexpected error that we would have to reconcile right?

Reply:
Yeah that's a good point, I've added an error message to double check the team name in https://github.com/isomerpages/isomer-conversion-scripts/pull/56/commits/877482372a73406d5f2d5605b32f7a5ec558744e